forte/processors/data_augment/algorithms/eda_processors.py

```diff
@@ -0,0 +1,243 @@
+    r"""
+    Data augmentation processor for the Random Swap operation.
+    Randomly choose two words in the sentence and swap their positions.
+    Do this n times.
+    """
+
+    def _process(self, input_pack: MultiPack):
+        augment_entry = get_class(self.configs["augment_entry"])
+        new_packs: List[Tuple[str, DataPack]] = []
+
+        for pack_name, data_pack in input_pack.iter_packs():
+            annotations = list(data_pack.get(augment_entry))
+            if len(annotations) > 0:
+                replace_map: Dict = {}
+                for _ in range(ceil(self.configs['alpha'] * len(annotations))):
+                    swap_idx = random.sample(range(len(annotations)), 2)
+                    new_idx_0 = swap_idx[1] if swap_idx[1] not in replace_map \
+                        else replace_map[swap_idx[1]]
+                    new_idx_1 = swap_idx[0] if swap_idx[0] not in replace_map \
+                        else replace_map[swap_idx[0]]
+                    replace_map[swap_idx[0]] = new_idx_0
+                    replace_map[swap_idx[1]] = new_idx_1
+                pid: int = data_pack.pack_id
+                for idx in replace_map:
+                    self.replaced_annos[pid]\
+                        .append((annotations[idx].span,
+                                 annotations[replace_map[idx]].text))
+
+            new_pack_name = "augmented_" + pack_name
+            new_pack = self.auto_align_annotations(
+                data_pack=data_pack,
+                replaced_annotations=self.replaced_annos[
+                    data_pack.meta.pack_id
+                ]
+            )
+            new_packs.append((new_pack_name, new_pack))
+
+        for new_pack_name, new_pack in new_packs:
+            input_pack.add_pack_(new_pack, new_pack_name)
+
+        self._copy_multi_pack_links(input_pack)
+
+    @classmethod
+    def default_configs(cls):
+        """
+        Returns:
+            A dictionary with the default config for this processor.
+            Additional keys for Random Swap:
+            - n: the number of times we perform the Random Swap operation
+        """
+        config = super().default_configs()
+        config.update({
+            'augment_entry': "ft.onto.base_ontology.Token",
+            'other_entry_policy': {
+                "entry": [
+                    "ft.onto.base_ontology.Document",
+                    "ft.onto.base_ontology.Sentence"
+                ],
+                "policy": ["auto_align", "auto_align"],
+            },
+            "kwargs": {},
+            'alpha': 0.1
+        })
+        return config
+
+
+class RandomInsertionDataAugmentProcessor(ReplacementDataAugmentProcessor):
+    r"""
+    Data augmentation processor for the Random Insertion operation.
+    Find a random synonym of a random word in the sentence that is
+    not a stop word. Insert that synonym into a random position in
+    the sentence. Do this n times.
```

Comment:
Again, there is probably no `n`. Please document the actual behavior of this class.